examples/water/se_atten/input.json

```diff
@@ -7,7 +7,7 @@
     ],
     "descriptor": {
       "type": "se_atten",
-      "stripped_type_embedding": true,
+      "stripped_type_embedding": false,
```

Comment:
new behavior `true` is preferred. @njzjz plz also check

Reply:
> new behavior `true` is preferred. @njzjz plz also check

Agree.